scripts/v2/transfer-feeds.ts

```diff
@@ -0,0 +1,84 @@
+import { ethers } from 'hardhat'
+import { market } from '../../api/market'
+import { ethGas } from '../../api/ethGas'
+import { addedFeeds } from './data/addedFeeds'
+import { feedTransfers } from './data/feedTransfers'
+import { convertToWei, exportFeedTransfers } from './utils/helpers'
+
+let network
+let newFeedTransfers = []
+
+const getFeedsToAdd = async (walletAddress, network) => {
+  const networkId = network.chainId === 7777 ? 1 : network.chainId
+  const excludeFeeds = addedFeeds[networkId].concat(feedTransfers)
+
+  const totalFeeds = (
+    await market.get('feeds', {
+      params: { networkId },
+    })
+  ).data.totalCount
+
+  let feedsToAdd = []
+  for (let i = 1; (i - 1) * 50 < totalFeeds; i++) {
+    let response = await market.get('feeds', {
+      params: { networkId, size: 50, page: i },
+    })
+
+    let feeds = response.data.data
+
+    feeds = feeds.filter(
+      (feed) =>
+        feed.type.name === 'OFFCHAIN_AGGREGATOR' &&
+        feed.oracleAddresses
```

Comment:
As discussed, the OCR transmitter will be within wallet addresses but we can't guarantee which one is the OCR transmitter address. It'd be better to just accept this as a param within the script.